src/analysis/ConfigurationsExt.cpp

```diff
@@ -304,6 +306,209 @@ namespace espressopp {
       delete [] ids;
     }
 
+    void ConfigurationsExt::gather_modified() {
```

Comment:
We don't need two. I can remove one and rename it. I kept it when I wrote the modified version.